src/results.rs

```diff
@@ -120,14 +112,15 @@ pub(crate) struct GetMoreResult {
 
 /// Describes the type of data store returned when executing
 /// [`Database::list_collections`](../struct.Database.html#method.list_collections).
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Default)]
 #[serde(rename_all = "camelCase")]
 #[non_exhaustive]
 pub enum CollectionType {
     /// Indicates that the data store is a view.
     View,
 
     /// Indicates that the data store is a collection.
+    #[default]
```

Comment:
this felt like a reasonable default; don't think the choice here is that critical, though, since these impls are for mocking